Rakefile

```diff
@@ -4,3 +4,18 @@
 require File.expand_path('../config/application', __FILE__)
 
 Rails.application.load_tasks
+
+# Create a new DB backup, and import it to the local Postgres DB
+task :db do
+  app = ENV["HEROKU_APP"] || "fastlane-refresher"
+  db_name = ENV["DB_NAME"] || "refresher"
+
+  puts "This script is going to drop your local database #{db_name} and fetch the database from heroku #{app}. Please confirm!"
+  STDIN.gets
```

Comment:
I agree that there's time, but I think the message printed out might convince someone to try to answer "n", but wind up dropping the database anyway.

If we don't want to add extra handling, I'd just change the message to end with something like "Quit now if that doesn't sound good!"